docs/upgrade.md

```diff
@@ -0,0 +1,93 @@
+## Example upgrade from 4.16.x to 4.16.y
+### Starting conditions:
+- A compatible seed image with target cluster is created. Seed image has `4.16.y` ocp version. [Both target cluster and seed image has configured a shared container partition for IBU.](https://docs.openshift.com/container-platform/4.17/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-partition.html)
```

Comment:
Use "OCP" in caps. Would it be better to use "latest" for the URL instead of "4.17"? Comment applies to other URLs as well.